AY2021S2-CS2103-T14-1/tp
# UI: Change layout and update icon

## What this does
This PR:
1. changes the layout of SunRez to give more visual space for data and feedback
2. updates the icon of SunRez to its logo (from the address book)
3. swaps the positions of the `Edit` and `Help` menu items

## How to test
1. Launch SunRez
2. Check the updated icon in the top-left of the window
3. Check the layout of SunRez. It should looks something like the image below

![image](https://user-images.githubusercontent.com/77471078/113086937-38368e00-9215-11eb-9b24-fb55f6ed2ff2.png)
